src/Features/Core/Portable/CodeFixes/CodeFixService.cs

```diff
@@ -240,42 +240,59 @@ internal partial class CodeFixService : ICodeFixService
             }
         }
 
-        public async Task<CodeFixCollection?> GetDocumentFixAllForIdInSpanAsync(
+        public Task<CodeFixCollection?> GetDocumentFixAllForIdInSpanAsync(
             Document document, TextSpan range, string diagnosticId, CodeActionOptions options, CancellationToken cancellationToken)
+            => GetDocumentFixAllForIdInSpanAsync(document, range, diagnosticId, DiagnosticSeverity.Hidden, options, cancellationToken);
```

Comment:
this should be removed from teh interface.  existing callers can either call the new method, or you can make this method into an extension method that forwards to the new method.